waltid-services/waltid-wallet-api/src/test/kotlin/E2EWalletTestBase.kt

```diff
@@ -1,7 +1,6 @@
 import id.walt.crypto.keys.KeyGenerationRequest
 import id.walt.crypto.keys.KeyType
-import id.walt.issuer.IssuanceExamples.issuerOnboardingRequestDefaultExample
-import id.walt.issuer.IssuerOnboardingResponse
+import id.walt.issuer.issuance.IssuerOnboardingResponse
 import id.walt.webwallet.db.models.AccountWalletListing
```

Comment:
since we're moving to a new e2e test structure, maybe this class can be removed and have the e2e tests added with another pr?